src/main/java/WayofTime/alchemicalWizardry/client/nei/NEIMeteorRecipeHandler.java

```diff
@@ -50,7 +54,7 @@ public CachedMeteorRecipe(MeteorParadigm meteor, ItemStack focusStack) {
                 int yPos = 37 + 18 * row;
 
                 List<String> tooltips = new ArrayList<>();
-                float chance = component.getChance() / totalMeteorWeight;
+                float chance = component.getChance() / totalComponentWeight * (float) (1 - fillerChance / 100.0);
```

Comment:
I would precompute:
```java
float fillerRatio = fillerChance / 100.0;
float componentRatio = 1 - fillerRatio;
```
outside of the loop, and then multiply chances by this.

Reply:
Addressed in [b5f55eb](https://github.com/GTNewHorizons/BloodMagic/pull/60/commits/b5f55eb79eadb8e225dff4fafff22f3db3a52a2f)